spec/lib/manageiq/api/common/headers_spec.rb

```diff
@@ -10,4 +10,12 @@
       expect { ManageIQ::API::Common::Headers.current = header_bad }.to raise_exception(ArgumentError)
     end
   end
+
+  context "get forwardable headers" do
+    it "x-rh-identity" do
+      ManageIQ::API::Common::Headers.current = header_good
+      hash = ManageIQ::API::Common::Headers.forwardable_headers
+      expect(hash['x-rh-identity']).to eq('abc')
```

Comment:
Should this test include `X-Request-ID` as well?

Reply:
You probably should also have a test that `:blah` is *not* included in the hash.